encoding/rs/utils.go

```diff
@@ -35,7 +44,9 @@ func ToFrArray(data []byte) ([]fr.Element, error) {
 	return eles, nil
 }
 
-// ToByteArray converts a list of Fr to a byte array
+// ToByteArray serializes a slice of fields elements to a slice of bytes.
+// The byte array is created by serializing each Fr element in big-endian format.
+// It is the reverse operation of ToFrArray.
```

Comment:
Technically not quite the reverse, since `ToFrArray` adds padding, yet this function doesn't remove any padding

Doesn't necessarily need to be changed, though

Reply:
[d3e3b88](https://github.com/Layr-Labs/eigenda/pull/1104/commits/d3e3b882609bdba557227ab5e7ee45721f7ed442)